src/modules/battery.rs

```diff
@@ -9,6 +9,12 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     const BATTERY_DISCHARGING: &str = "⇣";
     const BATTERY_THRESHOLD: f32 = 10.0;
 
```

Comment:
```suggestion
// TODO: Update when v1.0 printing refactor is implemented to only
// print escapes in a prompt context.
```

Could you leave this note in the code so that anyone doing the 1.0 refactor will be able to look back on this?